src/format/parsed.rs

```diff
@@ -299,6 +299,10 @@ impl Parsed {
     /// Gregorian year and ISO week date year can have their century number (`*_div_100`) omitted,
     /// the two-digit year is used to guess the century number then.
     pub fn to_naive_date(&self) -> ParseResult<NaiveDate> {
+        self.to_naive_date_inner(true)
+    }
+
+    fn to_naive_date_inner(&self, use_default: bool) -> ParseResult<NaiveDate> {
```

Comment:
What's the use case for this? I'd prefer not to do this.

Reply:
I added a `use_default` flag, so that we can require that a date which includes an offset does not accept a string without minutes, hours or days.

But `to_naive_date` is a public function, so I could not change the signature of that one and made an 'inner' method. It is also not my preference...